### PR TITLE
Comment style

### PR DESCRIPTION
**Pull request by amuamushu**
Centered the comment input form and styled the structure of each comment. Additionally, an input for the commenter's name added.

https://screenshot.googleplex.com/uWaaHmBHnZ1.png

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -212,16 +196,42 @@ function getMessageFromJSON(pageReloadBoolean) {
...
+ * Creates a p tag to store the given {@code text} inside the
+ * {@code container}.
+ * @return {object} Returns an p tag using
+ *     the given text.
+ */
+function appendPTagToContainer(text, container) {
+  const pTag = document.createElement(P_TAG);
+  pTag.innerText = text;
+  container.appendChild(pTag);
+
```
Comment (f-camp) [inline portfolio/src/main/webapp/script.js:231]:
```suggestion
```

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -212,16 +196,42 @@ function getMessageFromJSON(pageReloadBoolean) {
...
+  return pTag;
+}
```
Comment (f-camp) [inline portfolio/src/main/webapp/script.js:233]:
```suggestion
  container.appendChild(pTag);
  return pTag;
}
```

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -57,7 +57,6 @@ body {
   padding-left: 20px;
   padding-right: 20px;
   padding-top: 200px;
-  padding-bottom: 20px;
 }
```
Comment (f-camp) [inline portfolio/src/main/webapp/style.css:61]: Please order the declarations below.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/script.js]: Done!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/style.css:61]: I removed padding-bottom so it is no longer there for this pull request. Is that what you were referring to or is there something else?

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -221,9 +226,40 @@ button {
...
+
+.comment {
+  font-weight: bold;
+}
+
+.info {
+  display: flex;
+  justify-content: space-around;
+}
+
```
Comment (f-camp) [inline portfolio/src/main/webapp/style.css:262]: Order these?

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (f-camp) [reply portfolio/src/main/webapp/style.css:61]: Commented below, I meant the rest of the file below.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/style.css]: Done!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/style.css:58]: Done!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -43,11 +43,13 @@
   private static final String COMMENT_AMOUNT = "amount";
   private static final String COMMENT_TEXT = "text";
   private static final String COMMENT_TIMESTAMP = "timestamp";
+  private static final String COMMENT_NAME = "name";
+  
+  private static final String ANONYMOUS = "anonymous";
```
Comment (billie-cat) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: nit: consider renaming this to ANONYMOUS_AUTHOR

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -212,16 +196,41 @@ function getMessageFromJSON(pageReloadBoolean) {
...
+
+  liElement.appendChild(infoDivElement);
+  const textPElement = appendPTagToContainer(comment.text, liElement);
+  textPElement.className = COMMENT_CLASS;
 
   ulElement.appendChild(liElement);
 }
 
+/**
+ * Creates a p tag to store the given {@code text} inside the
```
Comment (billie-cat) [inline portfolio/src/main/webapp/script.js:222]:
```suggestion
 * Creates a <p> tag to store the given {@code text} inside the
```

Comment (billie-cat) [reply portfolio/src/main/webapp/script.js:222]: You can rephrase this as: Creates and returns a <p> tag. That way you can remove the @return section.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 5 above)
Comment (amuamushu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: done!

File: portfolio/src/main/webapp/script.js (hunk 6 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/script.js]: Done!
